production/scrapers/utah_statewide.R

```diff
@@ -1,17 +1,17 @@
 source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
-utah_statewide_check_date <- function(x, date = Sys.Date()){
-    z <- xml2::read_html(x)
+utah_statewide_check_date <- function(url, date = Sys.Date()){
+    base_html <- xml2::read_html(url)
```

Comment:
This page is tricky since it has numerous dates last updated for different sections. To make sure we're getting the date we want for the staff data scraped, I suggest this change:

    base_html <- xml2::read_html(url)
    
    p_headers <- base_html %>%
        rvest::html_nodes("p") %>% 
        rvest::html_text()
    
    ## looking for staff update date
    ## first look for where on the page this is
    staff_cases_idx <- p_headers %>%
        str_which(., "(?i)total confirmed staff cases") 
    
    ## limit dates that will come up in search results 
    p_headers[c(staff_cases_idx:length(p_headers))] %>%
        # rvest::html_nodes("p") %>% 
        # rvest::html_text() %>% 
        {.[str_detect(., "(?i)updated")]} %>% 
        first() %>% 
        str_split("updated") %>% 
        unlist() %>% 
        {.[str_detect(., "20")]} %>% # look for year 20xx
        lubridate::mdy() %>%
        error_on_date(date)

Reply:
Like Kentucky, I found that I didn't need `{.[str_detect(., "20")]}` otherwise, implemented as written, many thanks!